opencti-platform/opencti-front/src/private/components/settings/groups/GroupLine.tsx

```diff
@@ -0,0 +1,233 @@
+import React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import ListItemSecondaryAction from '@mui/material/ListItemSecondaryAction';
+import { CheckCircleOutlined, DoNotDisturbOnOutlined, KeyboardArrowRightOutlined, ReportGmailerrorred } from '@mui/icons-material';
+import Skeleton from '@mui/material/Skeleton';
+import { Link } from 'react-router-dom';
+import makeStyles from '@mui/styles/makeStyles';
+import { GroupLine_node$data } from '@components/settings/groups/__generated__/GroupLine_node.graphql';
+import { GroupingsLinesPaginationQuery$variables } from '@components/analyses/groupings/__generated__/GroupingsLinesPaginationQuery.graphql';
+import Tooltip from '@mui/material/Tooltip';
+import { useFormatter } from '../../../../components/i18n';
+import ItemIcon from '../../../../components/ItemIcon';
+import type { Theme } from '../../../../components/Theme';
+import { DataColumns } from '../../../../components/list_lines';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  item: {
+    paddingLeft: 10,
+    height: 50,
+  },
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    float: 'left',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 10,
+  },
+  goIcon: {
+    position: 'absolute',
+    right: -10,
+  },
+  itemIconDisabled: {
+    color: theme.palette.grey?.[700],
+  },
+}));
+
+interface GroupLineProps {
+  dataColumns: DataColumns
+  node: GroupLine_node$data
+  paginationOptions: GroupingsLinesPaginationQuery$variables
+}
+
+const GroupLineComponent: React.FC<GroupLineProps> = (props) => {
+  const { fd, t_i18n } = useFormatter();
+  const classes = useStyles();
+  const { dataColumns, node } = props;
+
+  return (
+    <ListItem
+      classes={{ root: classes.item }}
+      divider={true}
+      button={true}
+      component={Link}
+      to={`/dashboard/settings/accesses/groups/${node.id}`}
+    >
+      <ListItemIcon>
+        <ItemIcon type="Group" />
+      </ListItemIcon>
+      <ListItemText
+        primary={
+          <>
```

Comment:
question: This is the first time I see this empty <> in code, is it intended ?

Reply:
it's a react fragment, a virtual organization of component (it's interpreted as a single react node containing all the children, but at render time there is no actual tag (div or anything else).
`<>` is a shortcut to <React.Fragment>

Reply:
And it is used because here you can have only one child at root. So as we want multiple we use an empty React fragment as root and imbricate our multiple elements inside. The plus to use this instead of using a div is that by the end we don't add anything in the generated DOM